doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -185,14 +168,121 @@ Traceability to Reference Model.
+| `req.sec.ntw.02` | Networking | The Architecture **must** have all security logs stored in accordance with applicable security standard (national, regional), for example ISO27001. | | -->
+
 <p align="center"><b>Table 2-8:</b> OpenStack Security Requirements.</p>
 
+
+
+<a name="2.4"></a>
+### 2.4 Architecture and OpenStack Recommendations
+
+The requirements listed in this section are optional, viz. do not need to be implemented in conformant implementations.
+
+<a name="2.4.1"></a>
+2.4.1 General Recommendations
+
+| Ref # | sub-category | Description |  Traceability |
+|----|-------|---------------------|-------------|
+| `rec.gen.cnt.01` | Cloud nativeness | The Architecture **should** consist of stateless service components. However, where state is required it must be kept external to the component. | OpenStack consists of both stateless and stateful services where the stateful services utilize a database. For latter see "[Configuring the stateful services](https://docs.openstack.org/ha-guide/control-plane-stateful.html)"|
+| `rec.gen.cnt.02` | Cloud nativeness | The Architecture **should** consist of service components implemented as microservices that are individually dynamically scalable. | |
+| `rec.gen.scl.01` | Scalability | The Architecture **should** support policy driven auto-scaling. | This requirement is currently not addressed but will likley be supported through [Senlin](https://docs.openstack.org/senlin/pike/index.html), cluster management service. |
+| `rec.gen.rsl.02` | Resiliency | The Architecture **should** support resilient OpenStack service components that are not subject to `req.gen.rsl.01`. | |
+
+<p align="center"><b>Table 2-9:</b> General Recommendations</p>
+
+
+<a name="2.4.2"></a>
+2.4.2 Infrastructure Recommendations
+
+| Ref # | sub-category | Description |  Traceability |
+|----|-------|---------------------|-------------|
+| `rec.inf.com.02` | Compute | The Architecture **should** include industry standard hardware management systems at both HW device level (embedded) and HW platform level (external to device). | |
+| `rec.inf.com.03` | Compute | The Architecture **should** support symmetrical CPU multi-processing with shared memory access as well as multi-threading. | |
+| `rec.inf.stg.05` | Storage | The Architecture **should** be able to accommodate VNFs that store back into its image through use of hypervisor attached volumes. | **Wouldn't that be a Security Violation** |
+| `rec.inf.stg.08` | Storage | The Architecture **should** allow use of externally provided large archival storage for its Backup / Restore / Archival needs. | |
+| `rec.inf.stg.09` | Storage | The Architecture **should** make available all non-host OS / Hypervisor / Host systems storage as network-based Block, File or Object Storage for tenant/management consumption. | |
+| `rec.inf.ntw.04` | Network | The Architecture **should** support service function chaining. | **Is this VNFM/NFVO requirement** |
```

Comment:
Can we leave this out entirely until the networking discussions are concluded?

Reply:
@kedmison Since these are only recommendations, I am not sure if there is any harm in leaving them in place. Some of these may be lost as they may be not be the subject of the networking decisions.